zerver/tests/test_alert_words.py

```diff
@@ -145,23 +149,50 @@ def test_json_list_add(self) -> None:
         )
         self.assert_json_error(result, "alert_words[0] is too long (limit: 100 characters)")
 
+
     def test_json_list_remove(self) -> None:
         user = self.get_user()
         self.login_user(user)
 
-        result = self.client_post(
+        self.client_post(
             "/json/users/me/alert_words",
             {"alert_words": orjson.dumps(["one", "two", "three"]).decode()},
         )
-        response_dict = self.assert_json_success(result)
-        self.assertEqual(set(response_dict["alert_words"]), {"one", "two", "three"})
 
         result = self.client_delete(
-            "/json/users/me/alert_words", {"alert_words": orjson.dumps(["one"]).decode()}
+            "/json/users/me/alert_words",
+            {"alert_words": orjson.dumps(["one"]).decode()},
         )
         response_dict = self.assert_json_success(result)
         self.assertEqual(set(response_dict["alert_words"]), {"two", "three"})
 
+        from zerver.models import AlertWord
```

Comment:
I suspect the linter will complain about this not being at the top of the file.  
... particularly in that this assumes that this test runs before `test_reactivate_word` which I don't think pytest promises to be true :)